src/mlbstandings/web.py

```
@@ -7,21 +7,31 @@
 class Web:
     @staticmethod
     def retryable(code: int) -> bool:
-        return False  # TODO
+        return code in set([429, 500, 503])
```

Ruff (0.11.9)

Unnecessary list literal (rewrite as a set literal)

Rewrite as a set literal

(C405)